templates/default/flyingduck-site.xml.erb

```diff
@@ -3,13 +3,17 @@
 
   <hops>
     <namenodeServers><%= @nn_fqdn %>:<%= node['hops']['nn']['port'] %></namenodeServers>
```

Comment:
`@nn_fqdn` never assigned

Reply:
also below this are trust store locations and so on, which could be used instead of hardcoding it in the code. (to do this config_root could be used as already shown for getting metrics port)

Reply:
the flyingduck-site-xml(.erb) is not needed anymore and can be removed